utils/Felt.go

```diff
@@ -87,3 +90,43 @@ func FeltArrToBigIntArr(f []*felt.Felt) []*big.Int {
 	}
 	return bigArr
 }
+
+// StringToByteArrFelt converts string to array of Felt objects.
+//
+// Parameters:
+// - s: string/bytearray to convert
+// Returns:
```

Comment:
Can we add a comment about the structure of the returned felts? eg something along the lines of [number of felts, felts..., size of last felt]

Reply:
Lets add this link to the function docs

https://docs.starknet.io/architecture-and-concepts/smart-contracts/serialization-of-cairo-types/#serialization_of_byte_arrays

and make a note that the first element in the returned felt array is the number of 31-byte words in the data array, followed by the list of 31-byte words, then the pending word (less than 31-bytes), and finally the length of the pending word.

Thanks